### DISCUSSION TITLE
Viewport

### DISCUSSION THREAD

**Discussion by Shubham-EV** [fuma-nama/fumadocs#838] [Q&A] [answered]
In my laptop screen the width between sidebar-content and content-toc is optimal

![image](https://github.com/user-attachments/assets/ee0a654d-d852-49c6-aa29-299998b75c06)

However, in my PC the width is quite large

![image](https://github.com/user-attachments/assets/e208f0d6-69be-40d6-8050-05e7a6c0795f)

This [one](https://fumadocs.vercel.app/docs/ui/theme#layout-width) decreases overall max-width which I don't want.

Is their any way to decrease width (Extra space) in PC between sidebar-content and content-toc?

**Comment by fuma-nama** [comment]
Don't quite understand the question, you can simply set a new layout width with CSS variable or the plugin.

**Reply by Shubham-EV** [reply]
Check two images provided above then latter one has more width (PC) between sidebar-content and content-toc  (White rectangle on image).
I want this width to be same as in former image. Setting layout width is affecting overall width I just want to adjust width between sidebar-content and content-toc .

**Reply by Skreeauk** [reply, marked as answer]
you can change max width like this, if need to do it on bigger width than 2xl (1536px) then just customize your tailwind config. Or if this is confusing, then do css selector in your css file
![image](https://github.com/user-attachments/assets/eb28d1f9-653b-45f3-b6d2-a5d6ec370ed5)
unless fuma makes it so u can put classname directly on DocsPage

**Reply by fuma-nama** [reply]
You can adjust the max layout width until the gap looks good in your opinion.

![image](https://github.com/user-attachments/assets/70d06a0c-614d-4d5d-b49b-a08b0944db8b)
(from my extra large monitor)

**Reply by Shubham-EV** [reply]
> You can adjust the max layout width until the gap looks good in your opinion.
> 
> ![image](https://github.com/user-attachments/assets/70d06a0c-614d-4d5d-b49b-a08b0944db8b)
> (from my extra large monitor)
> 

what this is doing is changing position of sidebar and toc as well. I want them fixed in their respective sides. I just want to increase width of center part.

**Reply by Shubham-EV** [reply]
> you can change max width like this, if need to do it on bigger width than 2xl (1536px) then just customize your tailwind config. Or if this is confusing, then do css selector in your css file ![image](https://private-user-images.githubusercontent.com/74183099/365669945-eb28d1f9-653b-45f3-b6d2-a5d6ec370ed5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU4OTAwNDcsIm5iZiI6MTcyNTg4OTc0NywicGF0aCI6Ii83NDE4MzA5OS8zNjU2Njk5NDUtZWIyOGQxZjktNjUzYi00NWYzLWI2ZDItYTVkNmVjMzcwZWQ1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA5VDEzNDkwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhZTA0Yzc2OGZiODgzZDg2MzE5ZTdhN2EwM2VjNTNiZDZjYTBhNjM1Mjg2YmNlNzIwN2RjNTdmYTFjNDExNmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gH0vkmHaPNR1FaG0Dze3dXOKr12Vec2DIJtDXbSMM9w) unless fuma makes it so u can put classname directly on DocsPage

yeah its working thanks :)
